app/src/main/java/com/randomappsinc/studentpicker/home/NameListsAdapter.java

```diff
@@ -146,20 +160,19 @@ public void renameList() {
         public void deleteList() {
             showDeleteDialog(position);
         }
-    }
 
-    @Override
-    public View getView(int position, View view, ViewGroup parent) {
-        NameListViewHolder holder;
-        if (view == null) {
-            LayoutInflater vi = (LayoutInflater) context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);
-            view = vi.inflate(R.layout.names_list_item, parent, false);
-            holder = new NameListViewHolder(view);
-            view.setTag(holder);
-        } else {
-            holder = (NameListViewHolder) view.getTag();
+        @Override
+        public void onClick(View v) {
+            if (listItemClickListener != null) {
+                int position = getAdapterPosition();
+                if (position != RecyclerView.NO_POSITION) {
+                    listItemClickListener.onItemClick(position);
+                }
+            }
         }
-        holder.loadList(position);
-        return view;
+    }
+
+    public interface OnListItemClickListener {
```

Comment:
public interfaces should go at the top of the class, even above the member variables. The reason for this is that public interfaces are very important (they define the contract between 2+ classes), so they should be at the top for easy access.